shared/resources/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -113,12 +113,6 @@
       "idiom" : "ipad",
       "filename" : "appicon152x152.png",
       "scale" : "2x"
-    },
-    {
-      "size" : "83.5x83.5",
-      "idiom" : "ipad",
-      "filename" : "appicon167x167.png",
```

Comment:
The deleted file below?  Is it just in the wrong place, wrong reference?  The file exists....